rawhttp-core/src/main/java/rawhttp/core/RawHttp.java

```diff
@@ -276,7 +276,7 @@ public FramedBody getFramedBody(StartLine startLine, RawHttpHeaders headers) {
      * @param headers HTTP request's headers
      * @return true if the headers indicate the request should have a body, false otherwise
      */
-    public static boolean requestHasBody(RawHttpHeaders headers) {
+    public static boolean requestHasBody(RawHttpHeaders headers, RequestLine requestLine) {
```

Comment:
This is a breaking change. We need an overload without the `RequestLine` so old code continues working.

Reply:
You can add `@Deprecated` to the method without `RequestLine` so we can remove it in the future.